### PR TITLE
Setup file added for pypi

### PR DESCRIPTION
**Pull request by oakeyc**

### CODE REVIEW

File: azclishell/__init__.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-__version__ = '0.1.1a31'
+__version__ = '0.1.1c3'
```
Comment (derekbekoe) [inline azclishell/__init__.py]: The versions wouldn't be in sync again.

**Review by derekbekoe — CHANGES_REQUESTED** [review]
The setup.cfg looks good but you included a version change.

**Comment by oakeyc** [comment]
I wanted to increment to version 0.1.1c3

**Comment by derekbekoe** [comment]
@oakeyc but then the version in setup.py would also have to change right?

**Comment by oakeyc** [comment]
@derekbekoe changes made

**Review by derekbekoe — APPROVED** [review]

**Pull request merged**